ui/webpack.dev.ts

```diff
@@ -5,6 +5,7 @@ const webpack = require('webpack')
 const common = require('./webpack.common.ts')
 const path = require('path')
 const PORT = parseInt(process.env.PORT, 10) || 8080
+const PUBLIC = process.env.PUBLIC || undefined
```

Comment:
I'd be curious to know how `webpack` handles `undefined` property values. Also, if `process.env.PUBLIC` value isn't passed in, wouldn't it be `undefined` by default?

Reply:
best case scenario is that the object isn't assigned ANYTHING, but that makes the code hard to read (lots of if(condition){ out.prop = 'whatever' }), and undefined is less truthy than an empty string by 🔬 <- that much.
it handles the undefined value quite well in that it still serves files.